src/RestApi/StoreApi/Schemas/CartItemSchema.php

```diff
@@ -63,11 +63,20 @@ protected function get_properties() {
 				'context'     => [ 'view', 'edit' ],
 				'readonly'    => true,
 			],
+			'summary'             => [
+				'description' => __( 'A short summary (or excerpt from the full description) for the product in HTML format.', 'woo-gutenberg-products-block' ),
+				'type'        => 'string',
+				'context'     => [ 'view', 'edit' ],
+			],
 			'short_description'   => [
-				'description' => __( 'Product short description or excerpt from full description.', 'woo-gutenberg-products-block' ),
+				'description' => __( 'Product short description in HTML format.', 'woo-gutenberg-products-block' ),
+				'type'        => 'string',
+				'context'     => [ 'view', 'edit' ],
+			],
+			'description'         => [
+				'description' => __( 'Product full description in HTML format.', 'woo-gutenberg-products-block' ),
```

Comment:
It looks like `previews/cart-items.js` isn't matching the schema of a cart item response.  Probably should fix that in this pull? (I'm also okay with creating an issue to address it outside the pull too).